scripts/bootstrap_history.py

```diff
@@ -38,23 +39,24 @@ def extend(from_str, line):
     if len(argv) > 2:
         message = argv[2]
     elif not (ident.startswith("pr") or ident.startswith("issue")):
-        api_url = urlparse.urljoin(PROJECT_API, "commits/%s" % ident)
+        api_url = urljoin(PROJECT_API, "commits/%s" % ident)
         req = requests.get(api_url).json()
         commit = req["commit"]
         message = commit["message"]
         message = get_first_sentence(message)
     elif requests is not None and ident.startswith("pr"):
         pull_request = ident[len("pr"):]
-        api_url = urlparse.urljoin(PROJECT_API, "pulls/%s" % pull_request)
+        api_url = urljoin(PROJECT_API, "pulls/%s" % pull_request)
         req = requests.get(api_url).json()
+        print(api_url)
```

Comment:
Debug leftover?

Reply:
yep -- I had copied this over from ephemeris, so needs updating there as well.

Reply:
I've dropped this in a rebased commit, thanks!